WordPress/twentytwentyfive
# Add patterns: Testimonials

**Description**

Fixes #57, #58, #59

I've tackled all three of these in a single PR as working through them I realised they depend on one another.

- I've set the "default" quote block style with a border as seen in the 3 column layout. The "plain" styling removes this border/padding.
- The base block sizing/spacing is as per the 2 and 3 column layouts, the larger single testimonial has increased sizing/spacing applied to the block itself

This PR also registers the `Capsule` block style variation for `core/heading` blocks which is used in a few sections. It seems that a heading would be semantically correct in the locations where this is being used.

Couple of notes:

- ~~I'm not happy with the vertical alignment of the heading/quote. Unfortunately, `space-between` isn't an option on stack (only on row) which is a shame as this is all it would need to be perfect. Unless there's another way to do this I'm not aware of?~~ Edit: found this in the block toolbar!

- We're limited with how we can style the location part of the citation, so I'm using a line break and `subscript` to get this working. Not that intuitive to set up, but I think this is okay for a pre-made pattern?

- The Figma uses a different font just for the location part of the citation (instrument sans), I'm assuming this should be using manrope?

**Screenshots**

![image](https://github.com/user-attachments/assets/455c651e-640b-4813-ac83-756e621974f7)

<img src="https://github.com/user-attachments/assets/95c7ec21-2ce2-4f6e-b4b4-23755faf84bc" width="360">

**Testing Instructions**

1. Create a page
1. Add the patterns (found under 'testimonials')
1. Confirm that they looks like the designs (spacing/colors/etc.)
